### DISCUSSION TITLE
Mobile preview on templates

### DISCUSSION THREAD

**Discussion by heggland** [resend/react-email#690] [Ideas]
### Goals

1. mobile preview on the email templates

### Background

As we all know, more and more people are checking emails on their mobile devices. Having a good way to preview the email on a smaller screen would catch these issues more effectively. Sending out test mail to fix the problems and repeat isn't the best approach for mobile. 

### Proposal

- resize the preview area from desktop to mobile view.
- button called mobile
![Untitled](https://github.com/resendlabs/react-email/assets/26274036/cf124ef0-5fcf-40c1-9d99-3789b9eab42d)

- - alternative make the preview area fully responsive

**Comment by jbouhier** [comment]
Although not a current feature of react-email, we can already do this in with:
- Resizing browser's width
- Moving dev tools separator in vertical split
- Entering RWD / device mode in dev tools (Firefox: top-right icon, Chrome: top-left icon)
